girder/api/v1/notification.py

```diff
@@ -121,13 +121,13 @@ def streamGen():
     def listNotifications(self, params):
         user, token = self.getCurrentUser(returnToken=True)
         since = params.get('since')
+        timestamp = time.time()
 
         if since is not None:
             since = datetime.utcfromtimestamp(since)
         notifications = list(NotificationModel().get(user, since, token=token))
-        if not notifications:
-            timestamp = time.time()
-        else:
+
+        if notifications:
             timestamp = notifications[0]['updatedTime']
             for notification in notifications:
                 timestamp = max(timestamp, notification['updatedTime'])
```

Comment:
Lines 131-133 can be rewritten as simply:

`timestamp = max(n['updatedTime'] for n in notifications)`